docs/hyperopt.md

```diff
@@ -422,6 +424,8 @@ Stoploss: -0.37996664668703606
 
 If you are optimizing stoploss values, Freqtrade creates the 'stoploss' optimization hyperspace for you. By default, the stoploss values in that hyperspace can vary in the range -0.5...-0.02, which is sufficient in most cases.
 
+If you use legacy freqtrade HyperOpts class samples where `stoploss_space()` method was copied in each custom hyperopt file, simply remove it in order to utilize Stoploss hyperoptimization space generated by freqtrade by default.
```

Comment:
i am not sure about using "legacy" here - it's very difficult for a user reading this in 2 months to determine what "legacy" means.
(same applies above for ROI)

Maybe "if you have `stoploss_space()` in your custom hyperopt file, best remove it to use ... ?

Reply:
Reworded, pls review.